src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -154,6 +154,38 @@
  * &lt;module name=&quot;VisibilityModifier&quot;/&gt;
  * </pre>
  * <p>
+ * Example with default values:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ * private int myPrivateField1;
+ *
+ * int field1;               // violation, must have a visibility modifier
+ *
+ * protected String field2;  // violation, protected visibility is not allowed
+ *
+ * public int field3 = 42;   // violation, not static final, immutable, matching the pattern
+ * public long serialVersionUID = 1L;
```

Comment:
> // violation, not static final, immutable, matching the pattern

remove "matching pattern" (in all examples), it is not matching. serialVersionUID is matching pattern, that is why it is without violation.

Reply:
yeah, "not" goes back to the whole sentence.
"not static final, immutable, nor matching the pattern"
I see it's necessary to show the PATTERN PROPERATY.

Reply:
> nor matching the pattern

ok, lets use this wording to be more clear.

Reply:
Done